python/holders/src/constants/pools.json

```diff
@@ -96,5 +96,33 @@
     "0xFEBc0e1429f9A36D60dCe5eBF85EFFaa4a44051F",
```

Comment:
@prodesert22 instead of hard coding this, can we rather get this dynamically from the Subgraph?

Reply:
No why, I'm thinking of transforming this code for all tokens from all chains using the same uniswap factory system